service/src/main/java/bio/terra/catalog/app/App.java

```diff
@@ -27,6 +29,8 @@
       // Scan all service-specific packages beneath the current package
       "bio.terra.catalog"
     })
+@OpenAPIDefinition(info = @Info(title = "Terra", version = "2.0"))
```

Comment:
Is this needed? I don't see it used in the jade-data-repo sources.

Reply:
it was left over from me experimenting with learning notations. I will remove it if thats ok?